Houseypricey.js

```diff
@@ -0,0 +1,49 @@
+/*Your friend Peter is considering a house that is 8m wide, 10m deep and 10m high. 
+The garden size is 100m2. The house costs 2.500.000.*/
+
+//Peter's House Cost
+
+const houseWidth = 8;
+const houseDepth = 10;
+const houseHeight = 10;
+const gardenSizeInM2 = 100;
+const houseCost = 2500000;
```

Comment:
You can prefix those with `peter` just as you do with `julia` for better understanding of what they relate to.